src/libsyntax/parse/parser.rs

```diff
@@ -6001,8 +6002,8 @@ impl<'a> Parser<'a> {
             // single-variant-enum... :
             let m = Mac_ { path: pth, tts: tts, ctxt: EMPTY_CTXT };
             let m: ast::Mac = codemap::Spanned { node: m,
-                                             span: mk_sp(mac_lo,
-                                                         self.last_span.hi) };
+                                                    span: mk_sp(mac_lo,
+                                                                self.last_span.hi) };
```

Comment:
Some spacing in this file seems to have been haywire before and still haywire after...